### ISSUE TITLE
Colorfy CLI input argument ' --path' 

### ISSUE THREAD

**Issue by ChristophWWagner** [EMS-TU-Ilmenau/colorfy#7]
I propose to do two things:
 1. Refactor `-p`/` --path` command line argument to `-i / `--input`
 2. To not assume the file extension of the input filename automatically to `.json`

To the reasons for these proposals:
 1. For the first proposed change currently the usage states for `--path`: `path to the *.JSON input file without the file extension`. This basically means that a filename should be provided, not a path. I distinct between the two in that a path merely references to a certain directory whereas a file actually is some storage unit that can be input. Therefore I find the term to be misleading.
 2. Assuming the file extension prohibits the user from freely specifying the input file to use. Further, it complicates everything as soon as you write scripts that would batch-process files. Concluding with a more technical argument, assuming the extension also assumes the casing and since there are both, case-sensitive and case-insensitive file systems, the behaviour of the script could significantly differ just by case variations such as `.Json`, `.jSoN`, `.JSON` (which happen automatically on a regular basis when moving files between volumes that behave differently in that sense)

Please comment on this. If approved, I propose to relabel this issue *enhancement* and implement the change before pypi rollout.

**Comment by SebastianSemper** [comment]
I agree. -p is misleading and assuming the file-ending as well. same goes for all the other instances in the code, where this happens.
